pkg/service/did/web.go

```diff
@@ -36,7 +36,7 @@ var _ MethodHandler = (*webHandler)(nil)
 
 type CreateWebDIDOptions struct {
 	// e.g. did:web:example.com
-	DIDWebID string `json:"didWebId" validate:"required"`
+	DIDWebId string `json:"didWebId" validate:"required"`
```

Comment:
oh, this was correct, I'm not sure why your json showed up as `didWebID` maybe either is fine?

I would revert this

Reply:
yup linter caught this